src/traccuracy/matchers/_ctc.py

```diff
@@ -88,23 +87,29 @@ def _compute_mapping(
                 if pred_label_key in G_pred.graph.nodes[node]
             }
 
-            frame_map = match_frame_majority(gt_frame, pred_frame)
-            # Switch from segmentation ids to node ids
-            for gt_label, pred_label in frame_map:
-                mapping.append((gt_label_to_id[gt_label], pred_label_to_id[pred_label]))
-
-        return mapping
-
+            gt_boxes = np.asarray([gt.graph.nodes[node]["bbox"] for node in gt_frame_nodes])
+            pred_boxes = np.asarray([pred.graph.nodes[node]["bbox"] for node in pred_frame_nodes])
```

Comment:
This will fail if we didn't load the data through the ctc loader and therefore don't have the bboxes on the nodes. I think we can just check the first node for "bbox" before proceeding with trying to collect the bboxes. I'm guesing this same pattern is also present in the IOUMatcher?